### ISSUE TITLE
Install Content Manager 

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/documentation#477]
In order to improve accessibility to update the documentation a content manager should be installed.
After some research netlify CMS has been chosen.

[More information here: https://www.netlifycms.org/](https://www.netlifycms.org/)

This should be plugged to the documentation.

**Comment by guimachiavelli** [comment]
The @meilisearch/docs-team has discussed this today and decided this is not a feature we want to have at the moment.
